sky/packages/sky/lib/src/services/service_registry.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui_internals' as internals;
+
+import 'package:mojo_services/mojo/service_registry.mojom.dart';
+import 'package:mojo/core.dart' as core;
+
+ServiceRegistryProxy _initServiceRegistryProxy() {
+  core.MojoHandle serviceRegistryHandle = new core.MojoHandle(internals.takeShellProxyHandle());
+  if (!serviceRegistryHandle.isValid)
+    return null;
+  return new ServiceRegistryProxy.fromHandle(serviceRegistryHandle);
+}
+
+final ServiceRegistryProxy _serviceRegistryProxy = _initServiceRegistryProxy();
+final ServiceRegistry serviceRegistry = _serviceRegistryProxy?.ptr;
```

Comment:
It's equivalent to:

`_serviceRegistryProxy ==  null ? null : _serviceRegistryProxy.ptr`